model/contract/UsageControlComponentEnpoint.ttl

```diff
@@ -0,0 +1,44 @@
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix ids: <https://w3id.org/idsa/core/> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+# Classes
+# -------
+
+ids:UsageControlComponentEndpoint
+    a owl:Class;
+    rdfs:label "Usage Control Component Endpoint class"@en ;
+    rdfs:comment "Defined usage control component endpoints (i.e. PIP, PXP, etc.) in the IDS."@en;
+	.
+
+ids:PxpEndpoint
+    a owl:Class;
+    rdfs:subClassOf ids:UsageControlComponentEndpoint;
+    rdfs:label "pxp endpoint"@en;
+    rdfs:comment "The reference to the endpoint which operates a specific action, URI or reference to an associated PXP interface."@en;
+.
+
+ids:PipEndpoint
+    a owl:class;
+    rdfs:subClassOf ids:UsageControlComponentEndpoint;
+    rdfs:label "pip endpoint"@en;
+    rdfs:comment "Refers to an endpoint which provides a specific piece of information (URI/reference to an associated PIP interface). "@en;
+.
+
+# Properties
+# ----------
+
+ids:endpointDescription a owl:ObjectProperty;
+     rdfs:label "pxp interface description"@en;
```

Comment:
Now that this property has been generalized to apply to all usage control component endpoints, the `rdfs:label` of this property and of `ids:accessURI` should be generalized as well.

Reply:
Corrected.